web/src/components/njwds-extended/HelpButton.tsx

```diff
@@ -3,7 +3,11 @@ import { UnStyledButton } from "@/components/njwds-extended/UnStyledButton";
 import { useConfig } from "@/lib/data-hooks/useConfig";
 import { ReactElement } from "react";
 
-export const HelpButton = (): ReactElement => {
+interface Props {
+  onClick?: () => void;
```

Comment:
dust - since this button is only used in formation, have you considered defining the onclick within the component itself? The same set of analytics is fired for this button.

If you prefer the props, I strongly recommend making the props required and not optional. I would make it required, because, if we use this button elsewhere, and AC is forgotten, we'll technically be blocked. Or, if we are using this button else where, we could easily forget it.

Reply:
I did but I feel like the odds that we want to reuse this component going forwards is high, given the generic style and use and I think we would want to possibly reuse it in the future. 

Having hard coded analytics wouldn't work if we want to reuse it and be generic which would be better in my opinion. 

I'm okay making this required I suppose, but it kinda assumes that we will always have analytics associated with all intercom things, which is probably fine.

Reply:
Agreed with Faraz's recommendation. In favor of renaming the component to something like `FormationHelpButton` at least for now to ensure the analytics event doesn't fire if the component is used elsewhere.